draft-ietf-dnsop-svcb-https.md

```diff
@@ -843,6 +818,9 @@ addresses in response to the SvcDomainName query. Failure to use A and/or
 AAAA response addresses could negatively impact load balancing or other
 geo-aware features and thereby degrade client performance.
 
+Each decoded value in the value list SHALL be an IP address of the appropriate
+family in standard textual format {{!RFC5952}}.
```

Comment:
Similarly disallow escape processing for ipv4hints and ipv6hints.

Reply:
Done